### PR TITLE
Improvement/navigation/travel certificate

### PR DESCRIPTION
**Pull request by julia-andersson**
## [APP-XXX]

- Navigation for travel certificate from help center
- Does not work from profile yet since we haven't reworked profile

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/App/Sources/AppDelegate+Deeplinks.swift [hunk 1]
```diff
@@ -95,9 +95,9 @@ extension AppDelegate {
             deepLinkDisposeBag += ApplicationContext.shared.$hasFinishedBootstrapping.atOnce().filter { $0 }
                 .onValue { [weak self] _ in
                     self?.deepLinkDisposeBag.dispose()
-                    let vc = TravelInsuranceJourney.travelCertificateModally()
+                    //                    let vc = TravelInsuranceJourney.travelCertificateModally()
```
Comment (julia-andersson) [inline Projects/App/Sources/AppDelegate+Deeplinks.swift:98]: replace with TODO

File: Projects/App/Sources/AppDelegate+Deeplinks.swift [hunk 2]
```diff
@@ -95,9 +95,9 @@ extension AppDelegate {
...
                     let disposeBag = DisposeBag()
-                    disposeBag += fromVC.present(vc)
+                    //                    disposeBag += fromVC.present(vc)
```
Comment (julia-andersson) [inline Projects/App/Sources/AppDelegate+Deeplinks.swift:100]: same as above

File: Projects/Profile/Sources/Views/Screens/Profile/Profile.swift [hunk 3]
```diff
@@ -142,7 +142,8 @@ extension ProfileView {
             } else if case .openEuroBonus = action {
                 EuroBonusView.journey
             } else if case .openTravelCertificate = action {
-                TravelInsuranceJourney.travelCertificatePush()
+                //                TravelInsuranceJourney.travelCertificatePush()
```
Comment (julia-andersson) [inline Projects/Profile/Sources/Views/Screens/Profile/Profile.swift:145]: will be fixed for profile navigation

File: Projects/TravelCertificate/Sources/TravelCertificateNavigation.swift [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+enum TravelCertificateRouterActionsWithoutBackButton: Hashable {
+    case processingScreen
+}
+
+public struct TravelCertificateNavigation: View {
+    @StateObject private var travelCertificateNavigationVm = TravelCertificateNavigationViewModel()
+    @StateObject var router = Router()
+    private var canCreateTravelInsurance: Bool
+    private var infoButtonPlacement: ToolbarItemPlacement
+    private var openCoInsured: () -> Void
```
Comment (julia-andersson) [inline Projects/TravelCertificate/Sources/TravelCertificateNavigation.swift:31]: not sure if there is a better way to do this?

**Review by sladan-hedvig — COMMENTED** [review]
Removed store 🥳 
Also, it would be nice to ListScreen has a way to have
canAddTravelInsurance inside viewModel and get response from the API if it can add travel certificate
Maybe change try await self.service.getList() to return a list and a bool

File: Projects/TravelCertificate/Sources/TravelCertificateNavigation.swift (hunk 4 above)
Comment (sladan-hedvig) [reply Projects/TravelCertificate/Sources/TravelCertificateNavigation.swift:33]: This seems fine.

**Review by sladan-hedvig — APPROVED** [review]

File: Projects/TravelCertificate/Sources/TravelCertificateNavigation.swift [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+        )
+        .withDismissButton()
+        .configureTitle(L10n.TravelCertificate.cardTitle)
+    }
+
+    @ViewBuilder
+    private func start(with specifications: [TravelInsuranceContractSpecification]) -> some View {
+        if specifications.count > 1 {
+            /* TODO: CHANGE TO PUSH? */
+            showContractsList(for: specifications)
```
Comment (julia-andersson) [inline Projects/TravelCertificate/Sources/TravelCertificateNavigation.swift:101]: @sladan-hedvig this one was push before. Make more sense to have it full screen though?

**Pull request merged**